org.flatpak.am-okay.yml

```diff
@@ -0,0 +1,35 @@
+id: org.flatpak.am-okay
```

Comment:
this app-id is invalid. `org.flatpak` belong to the flatpak project. This is NOT part of it.

Reply:
How can I solve it ?

Reply:
@baldeuniversel https://docs.flathub.org/docs/for-app-authors/requirements#application-id

Reply:
This file is still using the old ID. Update this line and rename the file accordingly using the new ID.